requirements.txt

```diff
@@ -3,3 +3,4 @@ Flask-SocketIO
 adafruit-tcs34725
 flask-sqlalchemy
 Flask-Restless
```

Comment:
Jeżeli nie używamy `Flask-Restless` to powinniśmy zaktualizować `requirements.txt`